DEV-3/SequenceFib/SequenceFib/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace SequenceFib
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.Write("Please, enter number ");
+            String inputString = Console.ReadLine();
+            int numbFibonacci = int.Parse(inputString);
+            if (numbFibonacci < 0)
+            {
+                Console.WriteLine("I can't check this number");
+            }
+            int[] array = new int[numbFibonacci + 3];
```

Comment:
Why do you memorize all Fibonacci numbers in massive? use two temporal variables